src/quo2/components/calendar/__tests__/calendar_component_spec.cljs

```diff
@@ -0,0 +1,65 @@
+(ns quo2.components.calendar.--tests--.calendar-component-spec
+  (:require [quo2.components.calendar.calendar.view :as calendar]
+            [test-helpers.component :as h]
+            [cljs-time.core :as t]))
+
+(def start-date (t/date-time (t/year (t/now)) (t/month (t/now)) 5))
+(def end-date (t/date-time (t/plus start-date (t/days 2))))
+
+(h/describe "calendar component"
+  (h/test "default render of calendar component"
+    (h/render
+     [calendar/calendar
+      {:start-date start-date
+       :end-date   end-date}])
+    (-> (h/expect (h/query-by-translation-text "Mo"))
+        (h/is-truthy)))
```

Comment:
I think you don't need to use the -> operator here please see other uses of is-truthy, perhaps you can simplify here.
Also just confirm that you verified these tests actually work by changing the check and ensuring the result is also changing 👍

Reply:
with query-by-translation-text ,tests will fail if i directly use `(h/is-truthy (h/query-by-translation-text "Mo"))`, so i have to use it with ->

Reply:
Thanks Mohsen!